include/exadg/structure/time_integration/driver_quasi_static_problems.cpp

```diff
@@ -31,17 +31,17 @@ namespace Structure
 {
 template<int dim, typename Number>
 DriverQuasiStatic<dim, Number>::DriverQuasiStatic(
-  std::shared_ptr<Interface::Operator<Number>> operator_,
```

Comment:
we have a mixed syntax here in the library.
I am changing all
`variable_` to `variable_in`
only because if my IDE underlines the names, I cannot see the `_` anymore.
Skipping it is a bit confusing if one has member variables of the same name:
`variable(variable)`
or
`this->variable = variable;`
instead
`variable(variable_in)`
I can read and one can forget the `this->` or one gets an error if one by mistake writes
`variable = variable;`
---which would likely be a bug, since it has no effect on `this->variable`.